File: VSYard/VSYardLibrary/VSYardLibrary/Tags.fs
namespace VSYardNS

open System
open System.Collections.Generic
open System.Linq
open System.Text
open System.ComponentModel.Composition
open Microsoft.VisualStudio.Text
open Microsoft.VisualStudio.Text.Classification
open Microsoft.VisualStudio.Text.Editor
open Microsoft.VisualStudio.Text.Tagging
open Microsoft.VisualStudio.Utilities

type TokenTag () = class
    interface ITag 
    end

File: YaccConstructor/VSYard/Classifiers/Tags.cs
//using System;
//using System.Collections.Generic;
//using System.Linq;
//using System.Text;
//using System.ComponentModel.Composition;
//using Microsoft.VisualStudio.Text;
//using Microsoft.VisualStudio.Text.Classification;
//using Microsoft.VisualStudio.Text.Editor;
//using Microsoft.VisualStudio.Text.Tagging;
//using Microsoft.VisualStudio.Utilities;

////It is simple tags 
//namespace MyCompany.VSYard.Classifiers
//{

//    [Export(typeof(ITaggerProvider))]
//    [ContentType("yard")]
//    [TagType(typeof(VSYardNS.TokenTag))]
//    internal sealed class OokTokenTagProvider : ITaggerProvider
//    {

//        public ITagger<T> CreateTagger<T>(ITextBuffer buffer) where T : ITag
//        {
//            return null;//new VSYardNS.TokenTagger(buffer) as ITagger<T>;
//        }
//    }
//}
